src/extension/_locales/en/messages.json

```diff
@@ -633,6 +633,21 @@
   "user_switch": {
     "message": "Switch user"
   },
+  "v7_hint_cover": {
+    "message": "https://www.aem.live/tools/sidekick/en/v7-teaser.png"
+  },
+  "v7_hint_title": {
+    "message": "Introducing the new AEM Sidekick"
+  },
+  "v7_hint_description": {
+    "message": "A brand-new AEM Sidekick extension is now available in the Chrome Web Store. Switch now to upgrade your user experience!"
```

Comment:
Another option

A brand-new version of the AEM Sidekick extension is now available in the Chrome Web Store. Switch today for enhanced performance, new features, and a seamless authoring experience.